scrapy/utils/reqser.py

```diff
@@ -47,7 +50,11 @@ def request_from_dict(d, spider=None):
     eb = d['errback']
     if eb and spider:
         eb = _get_method(spider, eb)
-    return Request(
+    try:
```

Comment:
@elacuesta If I understood you right, what you're trying to say is that requests were stored to a disk queue by a separate project and were read by another. I don't know if that is sensible use case. My opinion here goes with Mikhail i.e fail in such a case.